_install_guide/dinghy.md

```diff
@@ -33,7 +33,9 @@ spinUIUrl:         https://spinnaker.your-company.io
 githubCredsPath:   /path/to/github-creds # credentials for github api (username:token)
 stashCredsPath:    /path/to/github-creds # credentials for stash api (username:token)
 stashEndpoint:     http://stash.mycompany.com/rest/api/1.0", # url where stash is running
-debugLevel:        info # one of: (debug, info, warn, error, fatal, panic)
+logging:
+    level: INFO  # one of: (DEBUG, INFO, WARN, ERROR, FATAL, PANIC)
+    # file: /var/log/dinghy.log  (This is optional, default is stdout i.e., goes to docker logs, remove comment if you want to log to a file)
```

Comment:
you'll prob need some indication that you'll need to mount that volume to the host ec2 instance using `docker-compose-override.yml`

Reply:
That would make it specific to AWS install. Also, not all customers want to log to a mounted volume.

Reply:
For those that are using docker, how are they going to get the log files, wouldn't the logs be only on the container? Any log forwarder will need to be attached the dinghy's fs too.